iris-logo/README.md

```diff
@@ -0,0 +1,41 @@
+# The Iris Logo
+
+## Logo
+
+The official logo can be obtained, in SVG form, via
+
+    latexmk logo
```

Comment:
When I run this command, it creates a PDF file, but I cannot find a PNG file in the `iris-logo` folder. Am I doing something wrong?

Reply:
None of the commands I wrote in the README are supposed to produce a PNG.
The end result of both is a SVG.
With `latexmk logo` you get `logo.dvi` first (compiled with `latex`) and then `logo.svg` produced from the DVI using `dvisvgm` (included in TeXLive).
With `latexmk -pdf logo` you run `pdflatex` to produce `logo.pdf` and then `logo.svg` is generated from the PDF using `pdf2svg`.
I prefer the first option, but the previous setup was following the second so I included both. Having a PDF could also be handy for inclusion in LaTeX documents.
Note that the SVG produced by the two workflows would differ.

Reply:
Sorry, I meant SVG. I didn't get one of those either.

Reply:
Here's the folder content after `latexmk logo`:
```
design  favicon.tex  latexmkrc  logo.aux  logo.fdb_latexmk  logo.fls  logo.log  logo.pdf  logosetup.tex  logo.tex  README.md
```
There's no dvi file, but there is a pdf file.

Reply:
that is weird, maybe you have a global latexmkrc file making default to -pdf?
Maybe try with `latexmk -r latexmkrc logo`?

Reply:
(or `latexmk -dvi logo`)

Reply:
latexmk says
```
Rc files read:
  /etc/LatexMk
  ./latexmkrc
```
and the one in `/etc` contains
```
$pdf_mode = 4;
```
I am not sure what "4" means but yes it sounds like I have a PDF default. (Which makes sense, why would I want dvi files?^^) I don't remember if I set that or if this is a distro thing.

Reply:
> latexmk -r latexmkrc logo

That still does the same thing.

Maybe we should put `$pdf_mode = X` into the local `latexmkrc`, for the right value of `X`?

Reply:
`latexmk -dvi logo` says thins that end in
```
Rule 'dvipdf': File changes, etc:
   Changed files, or newly in use since previous run(s):
      'logo.dvi'
   Rule(s) that have been run and require run of dependent rule:
      'latex' which requires run of 'dvipdf'
------------
Run number 1 of rule 'dvipdf'
------------
For rule 'dvipdf', running '&do_viewfile(  )' ...
------------
Running 'dvipdf  "logo.dvi" "logo.pdf"'
------------
Latexmk: All targets (logo.dvi logo.pdf) are up-to-date
```
This doesn't mention an svg file, but one is still being created. :)